migrations/versions/e65ffbc90def_.py

```diff
@@ -0,0 +1,32 @@
+"""empty message
+
+Revision ID: e65ffbc90def
+Revises: 605a688d0cd2
+Create Date: 2022-04-18 11:29:29.984378
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'e65ffbc90def'
+down_revision = '77fdc857c94f'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('systems', sa.Column('cpu_state', sa.ARRAY(sa.String()), nullable=True))
+    op.add_column('systems', sa.Column('io_state', sa.ARRAY(sa.String()), nullable=True))
+    op.add_column('systems', sa.Column('memory_state', sa.ARRAY(sa.String()), nullable=True))
```

Comment:
```suggestion
    op.add_column('systems', sa.Column('cpu_states', sa.ARRAY(sa.String()), nullable=True))
    op.add_column('systems', sa.Column('io_states', sa.ARRAY(sa.String()), nullable=True))
    op.add_column('systems', sa.Column('memory_states', sa.ARRAY(sa.String()), nullable=True))
```
These fields are of Array type and can include multiple values so does it make more sense to keep names as `cpu_states/memory_states/io_states`

Reply:
@kgaikwad Thanks! Changes :rocket: